packages/dx-react-scheduler-demos/src/demo-sources/scheduler-featured-redux-integration/material-ui/demo.jsx

```diff
@@ -0,0 +1,330 @@
+import * as React from 'react';
+import { createStore } from 'redux';
+import { connect, Provider } from 'react-redux';
+import Paper from '@material-ui/core/Paper';
+import TextField from '@material-ui/core/TextField';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+import { teal } from '@material-ui/core/colors';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import { ViewState } from '@devexpress/dx-react-scheduler';
+import {
+  Scheduler,
+  WeekView,
+  Toolbar,
+  DateNavigator,
+  Appointments,
+  DayView,
+  ViewSwitcher,
+} from '@devexpress/dx-react-scheduler-material-ui';
+
+import { appointments } from '../../../demo-data/appointments';
+
+const styles = ({ spacing, palette }) => ({
+  flexibleSpace: {
+    margin: '0 auto 0 0',
+    display: 'flex',
+    alignItems: 'center',
+  },
+  textField: {
+    width: '100px',
+    marginLeft: spacing(1),
+    marginTop: 0,
+    marginBottom: 0,
+    height: spacing(4.875),
+  },
+  locationSelector: {
+    marginLeft: spacing(1),
+    height: spacing(4.875),
+  },
+  button: {
+    paddingLeft: spacing(0.5),
+    paddingRight: spacing(0.5),
+    width: spacing(10),
+  },
+  selectedButton: {
+    paddingLeft: spacing(0.5),
+    paddingRight: spacing(0.5),
+    width: spacing(10),
+    background: teal[400],
+    color: teal[50],
+    '&:hover': {
+      backgroundColor: teal[500],
+    },
+    border: `1px solid ${teal[400]}!important`,
+    borderLeft: `1px solid ${teal[50]}!important`,
+    '&:first-child': {
+      borderLeft: `1px solid ${teal[50]}!important`,
+    },
+  },
+  title: {
+    fontWeight: 'bold',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+  },
+  textContainer: {
+    lineHeight: 1,
+    whiteSpace: 'pre-wrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    width: '100%',
+  },
+  time: {
+    display: 'inline-block',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+  },
+  text: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+  },
+  container: {
+    width: '100%',
+  },
+  weekendCell: {
+    backgroundColor: fade(palette.primary.main, 0.1),
+    '&:hover': {
+      backgroundColor: fade(palette.primary.main, 0.14),
+    },
+    '&:focus': {
+      backgroundColor: fade(palette.primary.main, 0.16),
+    },
+  },
+  firstRoomAppointment: {
+    backgroundColor: '#26A69A',
+    '&:hover': {
+      backgroundColor: '#26A69A',
+    },
+    '&:focus': {
```

Comment:
Is `focus` necessary? I think no.

Also, we can use here teal[400] and teal[500] instead of strings